core/src/main/scala/org/apache/spark/status/api/v1/ApplicationListResource.scala

```diff
@@ -53,12 +55,7 @@ private[v1] class ApplicationListResource(uiRoot: UIRoot) {
           attempt.startTime.getTime <= maxDate.timestamp
```

Comment:
I agree. Compared to the transformation from ApplicationHistoryInfo to ApplicationInfo, this one is negligible.